### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by fflorent**
## Problem

- The README file is not generated
- The app has never been tested by the CI

## Solution

- Let the bot do this job

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by fflorent** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/remove
Comment (fflorent) [inline scripts/remove:1]: Why removing ynh_redis?

Comment (ericgaspar) [reply scripts/remove:1]: Sorry, my mistake, I moved the helper to `_common.sh` but somehow I didn't save.

Comment (fflorent) [reply scripts/remove:1]: Ok, np

**Comment by fflorent** [comment]
!testme

**Comment by fflorent** [comment]
!testme

**Pull request merged**